store/tikv/coprocessor.go

```diff
@@ -358,13 +356,14 @@ func (it *copIterator) work(ctx goctx.Context, taskCh <-chan *copTask) {
 }
 
 func (it *copIterator) run(ctx goctx.Context) {
+	taskCh := make(chan *copTask)
```

Comment:
Does this hurt the performance of normal queries?

Reply:
It's the same as before, I just move it from `copIterator` to a local variable.

Reply:
But the copIterator member `taskCh` is initialized with concurrency capacity.

Reply:
Yeah, that's a core change as the description:

> 2. make coprocessor task channel unbuffered (reduce memory use)

We already have N workers fetch data concurrently, if the `taskCh` is a buffered one with size N,
then pending coprocessor response would be actually 2*N.

What's worse, we're handling response in a streaming style (calling `Next()` function), the memory occupation amplification will be much more than 2*N.

Reply:
This was configurable before, why not keep it configurable but set the default value to 0?